rclcpp/test/rclcpp/test_timer.cpp

```diff
@@ -364,3 +386,77 @@ TEST_P(TestTimer, test_timer_without_autostart)
     std::chrono::nanoseconds::max().count());
   EXPECT_FALSE(timer_without_autostart->is_canceled());
 }
+
+/// Timer testing bring up and teardown
+class TestTimerWithInfo : public ::testing::TestWithParam<TimerType>
```

Comment:
what is this class for? never be used for anywhere?

Reply:
good catch, removed...